rpcs3/rpcs3qt/recvmessage_dialog_frame.cpp

```diff
@@ -85,6 +85,8 @@ bool recvmessage_dialog_frame::Exec(SceNpBasicMessageMainType type, SceNpBasicMe
 		{ accept_or_deny(SCE_NP_BASIC_MESSAGE_ACTION_ACCEPT); });
     connect(btn_deny, &QAbstractButton::clicked, this, [&accept_or_deny]()
 		{ accept_or_deny(SCE_NP_BASIC_MESSAGE_ACTION_DENY); });
+    connect(btn_cancel, &QAbstractButton::clicked, this, [m_dialog=this->m_dialog]()
```

Comment:
Weird formatting, just capture [this].
Or just connect it to the cancel slot of m_dialog